Simplenote/Classes/SPNoteHistoryViewController.swift

```diff
@@ -0,0 +1,179 @@
+import UIKit
+
+// MARK: - SPNoteHistoryViewController: Shows history for a note
+//
+class SPNoteHistoryViewController: UIViewController {
+    @IBOutlet private weak var dateLabel: UILabel!
+    @IBOutlet private weak var errorMessageLabel: UILabel!
+    @IBOutlet private weak var slider: SPSnappingSlider!
+    @IBOutlet private weak var restoreButton: UIButton!
+    @IBOutlet private weak var activityIndicator: UIActivityIndicatorView!
+
+    private let controller: SPNoteHistoryController
+    private var items: [SPNoteHistoryController.Presentable] = []
+
+    /// Designated initialize
+    ///
+    /// - Parameters:
+    ///     - controller: business logic controller
+    ///
+    init(controller: SPNoteHistoryController) {
+        self.controller = controller
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    @available(*, unavailable)
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        styleDateLabel()
+        styleErrorMessageLabel()
+        styleSlider()
+        styleRestoreButton()
+        styleActivityIndicator()
+
+        listenForSliderValueChanges()
+
+        controller.observer = { [weak self] state in
+            self?.update(with: state)
+        }
+        controller.onViewLoad()
+
+        trackScreen()
+    }
+}
+
+// MARK: - Private Methods
+//
+private extension SPNoteHistoryViewController {
+    func styleDateLabel() {
+        dateLabel.textColor = .simplenoteNoteHeadlineColor
+    }
+
+    func styleErrorMessageLabel() {
+        errorMessageLabel.textColor = .simplenoteTextColor
+    }
+
+    func styleSlider() {
+        let color = UIColor.simplenoteGray50Color.withAlphaComponent(0.2)
```

Comment:
We should probably move this to a UIColor property (or drop the 0.2 inline!)

Reply:
That's taken care of in the next PR (#796), where I have extracted this to a UIColor property :-)

Reply:
Thank you!